src/expansion/mod.rs

```diff
@@ -39,6 +39,9 @@ pub struct Options {
 	/// Sets the processing mode.
 	pub processing_mode: ProcessingMode,
 
+	// If true, an error is returned if a value fails to expand. If false, the value is dropped.
+	pub strict_expansion: bool,
```

Comment:
Since we already are in the `expansion` module, in the expansion options, I don't think we can get rid of the `_expansion` suffix in the field name.

Reply:
Updated.